static/app/constants/index.tsx

```diff
@@ -309,6 +309,14 @@ export const DATA_CATEGORY_INFO = {
     titleName: t('Monitor Check-Ins'),
     uid: 10,
   },
+  [DataCategoryExact.MONITOR_SEAT]: {
+    name: DataCategoryExact.MONITOR_SEAT,
+    apiName: 'monitorSeat',
+    plural: 'monitorSeats',
```

Comment:
I think thi sshould be `monitor seats`

Reply:
or maybe `cron monitors`

Reply:
I believe we want "cron monitor(s)" for anything that gets displayed in the UI

Reply:
ahh I talked with tommy/vu and I think we're changing it to just Monitors to keep it consistent with other items in the checkout flow

Reply:
going to change this to cron monitors and merge it in, will change to monitors on BE/FE if needed after syncing more up with Tommy